### DISCUSSION TITLE
Help! Upload excel  .etc fail!!!

### DISCUSSION THREAD

**Discussion by AI-Beans** [open-webui/open-webui#1279] [General]
When I want to upload an excel file, an error is reported. The information is as follows:
![b006760d20704ccf075b2d379a0e93d](https://github.com/open-webui/open-webui/assets/58964439/ad8487f5-eada-4dc0-acc3-9209bfed553e)
![image](https://github.com/open-webui/open-webui/assets/58964439/5687a978-3d25-4ff7-bef2-f23077174a55)

**Comment by catbor** [comment]
Yes, when I uploaded the excel file, I encountered the same problem, hopefully this can be fixed to make the knowledge base more flexible!
![image](https://github.com/open-webui/open-webui/assets/59753214/57c9c3bd-df32-4b98-a06c-be918b51b6ea)

**Comment by krishna-santosh** [comment]
I faced the same issue with .xls file, converted it to .csv and things worked fine after that.

**Comment by jiangbophd** [comment]
it is still same issue, is already fixed？

**Comment by jiangbophd** [comment]
it is still same issue, is already fixed？

**Reply by kaiiquetome** [reply]
Try to use Docling https://docs.openwebui.com/features/document-extraction/docling
